tests/test_quixstreams/test_app.py

```diff
@@ -748,12 +748,13 @@ def get_cfg_builder(quix_sdk_token):
         ) as consumer_init_mock, patch(
             "quixstreams.app.RowProducer"
         ) as producer_init_mock:
-            Application(
+            app = Application(
                 consumer_group=consumer_group,
                 quix_sdk_token=quix_sdk_token,
                 consumer_extra_config=extra_config,
                 producer_extra_config=extra_config,
             )
+            assert app._is_quix_app
```

Comment:
Hey, please avoid calling the protected variables outside of the instance methods.

Is there another way to verify it works without checking `._is_quix_app`?